src/main.go

```diff
@@ -39,9 +40,12 @@ import (
 
 // @securityDefinitions.basic BasicAuth
 func main() {
-
 	fmt.Println("")
 
+	// giving a default value of "1323"
+	port := flag.String("port", "1323", "port number for the restapiserver to listen to")
+	flag.Parse()
+
```

Comment:
How about adding validation for the given port number? [1-65535]

If you agree, you can refer to the code below.
I tested following code myself :) 


```suggestion
	// giving a default value of "1323"
	port := flag.String("port", "1323", "port number for the restapiserver to listen to")
	flag.Parse()

	// validate arguments from flag
	validationFlag := true
	// validation: port
	// return TRUE if your number is in [1-65535] range
	if portInt, err := strconv.Atoi(*port); err == nil {
		if portInt < 1 || portInt > 65535 {
			validationFlag = false
		}
	} else {
		validationFlag = false
	}
	if !validationFlag {
		fmt.Printf("%s is not a valid port number.\n", *port)
		fmt.Printf("Please retry with a valid port number (ex: -port=[1-65535]).\n")
		os.Exit(1)
	}
```

```bash
$ ./cb-tumblebug -port=23242334

23242334 is not a valid port number.
Please retry with a valid port number (ex: -port=[1-65535]).
```

```bash
$ ./cb-tumblebug -port=123NiceContributor

123NiceContributor is not a valid port number.
Please retry with a valid port number (ex: -port=[1-65535]).
```